cypress/integration/settings/circulation/general/edit-circulation-rules.spec.js

```diff
@@ -0,0 +1,155 @@
+import uuid from 'uuid';
+import { kebabCase } from 'lodash';
+
+import { getTestEntityValue } from '../../../../support/utils/stringTools';
+import TestType from '../../../../support/dictionary/testTypes';
+import SettingsMenu from '../../../../support/fragments/settingsMenu';
+import {
+  CodeMirror,
+  CodeMirrorHint,
+  Button,
+} from '../../../../../interactors';
+import InteractorsTools from '../../../../support/utils/interactorsTools';
+
+describe('ui-circulation-settings: Edit circulation rules', () => {
+  const calloutMessages = {
+    CIRCULATION_RULES_UPDATE_SUCCESS: 'Rules were successfully updated.',
+  };
+  const newNoticePolicyData = {
```

Comment:
move such test data into related fragments

Reply:
Do you mean just move it to fragment files and import these data from them?

Reply:
yes